tools/StaticAnalysis/Exceptions/Az.FrontDoor/SignatureIssues.csv

```diff
@@ -1,13 +1,14 @@
-"Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewAzureRmFrontDoorWafManagedRuleObject","New-AzFrontDoorWafManagedRuleObject","1","8100","New-AzFrontDoorWafManagedRuleObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewAzureRmFrontDoorWafMatchConditionObject","New-AzFrontDoorWafMatchConditionObject","1","8100","New-AzFrontDoorWafMatchConditionObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewAzureRmFrontDoorRoutingRuleObject","New-AzFrontDoorRoutingRuleObject","1","8100","New-AzFrontDoorRoutingRuleObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewAzureRmFrontDoorWafRuleGroupOverrideObject","New-AzFrontDoorWafRuleGroupOverrideObject","1","8100","New-AzFrontDoorWafRuleGroupOverrideObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
-"Microsoft.Azure.Commands.FrontDoor.dll","Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewAzureRmFrontDoorWafManagedRuleOverrideObject","New-AzFrontDoorWafManagedRuleOverrideObject","1","8100","New-AzFrontDoorWafManagedRuleOverrideObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
\ No newline at end of file
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
+Microsoft.Azure.Commands.FrontDoor.dll,Microsoft.Azure.Commands.FrontDoor.Cmdlets.NewAzureRmFrontDoorBackendObject,New-AzFrontDoorBackendObject,1,8100,New-AzFrontDoorBackendObject Does not support ShouldProcess but the cmdlet verb New indicates that it should.,Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue
```

Comment:
@isra-fel The builds are failing on static analysis for the preexisting cmdlet New-AzFrontDoorBackendObject with signature issue: "Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue" However, when I check the exceptions file, this cmdlet is already included. I'm not sure how to resolve this build issue. Please advise!

Reply:
You should edit the csv with a text editor... If you look at the change you'll see there's supposed to be quotemarks in the both ends of the words